doc/database.md

```diff
@@ -0,0 +1,142 @@
+
+# Database #
+
+## PostgreSQL ##
+
+If you want use `PostgreSQL` as database engine:
+
+    sudo apt install postgresql-server-dev-all postgresql postgresql-client
+
+### Minimalist introduction to PostgreSQL ###
+
+If you already use/know `PostgreSQL`, you can directly go to *Test the database access*.
+
+#### Driver ####
+
+Tracim uses the `psycopg` driver between the `SQLAlchemy` ORM and the `PostgreSQL` RDBMS. Run the following command to install the right version:
+
+    pip install -r install/requirements.postgresql.txt
+
+#### Allowing local connections on PostgreSQL ####
+
+Debian `PostgreSQL` stores connections authorization in `/etc/postgresql/9.1/main/pg_hba.conf`. Edit this file and check that connection from `127.0.0.1` are allowed using user/password. You should find the following line in the file:
+
+    # IPv4 local connections:
+    host    all             all             127.0.0.1/32            md5
+
+If you changed the file, reload `PostgreSQL`:
+
+    service postgresql reload
+
+#### Creating a user and associated database ####
+
+You need a database and associated user/password. Tracim comes with `pgtool`, tool that will make this step easy:
+
+    ./bin/pgtool help
+
+Run the following self explanatory commands:
+
+    sudo su postgres
+    ./bin/pgtool create_user tracimuser tracimpassword
```

Comment:
The database documentation page presents, as is, one main benefit:
- the ease for the reader to reproduce the command by simply copying/pasting the considered lines.

However, we should warn our reader to use this commands only for a development environment.